classes-metodos-atributos/exercicio-07/src/main/java/MyProgram.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Scanner;
+
+public class MyProgram {
+
+    public static void main(String[] args) {
+
+        Scanner entrada = new Scanner(System.in);
```

Comment:
Mantenha o padrão, se em um local está chamando de in, chame todos de in. Se chamar de entrada, chame todos de entrada.

Reply:
Essa é a mensagem que está na aba PROBLEMAS/PROBLEMS do VSCode: Resource leak: 'in' is never closed.
Então, sim, precisa ser fechada e o código precisa vir sem nenhum PROBLEMA relatado nessa aba. 
Faça as verificações antes de enviar.